cmd/genji/shell/command.go

```diff
@@ -24,3 +29,62 @@ func runTablesCmd(db *genji.DB) error {
 
 	return nil
 }
+
+// runTableIndexCmd prints the all indexes that the given table contains.
+func runTableIndexCmd(db *genji.DB, tableName string) error {
+	err := db.ViewTable(tableName, func(tx *genji.Tx, table *database.Table) error {
+		return table.PrintIndexes()
+	})
+
+	if err == document.ErrFieldNotFound {
+		return nil
+	}
+
+	return err
+}
+
+// runAllIndexesCmd shows all indexes that the database contains.
+func runAllIndexesCmd(db *genji.DB) error {
+	var tables []string
+	err := db.View(func(tx *genji.Tx) error {
+		var err error
+		tables, err = tx.ListTables()
+		return err
+	})
+
+	if err != nil {
+		return err
+	}
+
+	for _, table := range tables {
+		err = db.View(func(tx *genji.Tx) error {
+			t, err := tx.GetTable(table)
+			if err != nil {
+				return err
+			}
+
+			return t.PrintIndexes()
+		})
+
+		//
+		if err == database.ErrTableNotFound {
+			return err
+		}
+	}
+
+	return nil
+}
+
+// runIndexesCmd executes all indexes of the database or all indexes of the given table.
```

Comment:
```suggestion
// runIndexesCmd executes the .indexes command.
```

Reply:
@tzzed this comment has been marked as resolved but the suggestion is not applied. Probably due to an omission. Do you think you can apply it?